README.md

```diff
@@ -2,7 +2,7 @@
 
 [![Telegram](https://img.shields.io/static/v1?label=Telegram&message=RU&color=0088CC)](https://t.me/kaspresso)
 [![Telegram](https://img.shields.io/static/v1?label=Telegram&message=EN&color=0088CC)](https://t.me/kaspresso_en)
-[![master-documentation](https://github.com/KakaoCup/Kakao/actions/workflows/master-guthub-pages.yml/badge.svg)](https://github.com/KakaoCup/Kakao/actions/workflows/master-guthub-pages.yml)
+[![master-documentation](https://github.com/KakaoCup/Kakao/actions/workflows/master-github-pages.yml/badge.svg)](https://github.com/KakaoCup/Kakao/actions/workflows/master-guthub-pages.yml)
```

Comment:
Please fix in both places 
`[![master-documentation](https://github.com/KakaoCup/Kakao/actions/workflows/master-github-pages.yml/badge.svg)](https://github.com/KakaoCup/Kakao/actions/workflows/master-guthub-pages.yml)`